tests/test.rs

```diff
@@ -532,8 +563,8 @@ fn gnu_apple_sysroot() {
         test.shim("fake-gcc")
             .gcc()
             .compiler("fake-gcc")
-            .target(&target)
-            .host(&target)
+            .target(target)
+            .host(target)
             .file("foo.c")
             .compile("foo");
 
```

Comment:
Documentation on `compile` should also be updated to reflect the new renaming rules.